tests/configuration/test_tentacle_configuration.py

```diff
@@ -83,9 +85,30 @@ async def test_factory_reset_config():
     _cleanup()
 
 
+async def test_fill_tentacle_config():
+    async with aiohttp.ClientSession() as session:
+        await api.install_all_tentacles(_tentacles_local_path(), aiohttp_session=session)
+
+    setup_config = configuration.TentaclesSetupConfiguration()
+    available_tentacle = util.load_tentacle_with_metadata(constants.TENTACLES_PATH)
+    await setup_config.fill_tentacle_config(available_tentacle, constants.TENTACLE_CONFIG_FILE_NAME)
```

Comment:
should we also test a success case (something else than `constants.TENTACLE_INSTALLATION_CONTEXT_OCTOBOT_VERSION_UNKNOWN`) to ensure it works ?

Reply:
By mocking octobot_constants.Version call ?

Reply:
U guess we can mock `_get_installation_context_bot_version`

Reply:
That's true :+1: